hbw/app/javascript/packs/hbw/components/entity_task.js.jsx

```diff
@@ -1,22 +1,19 @@
 import { withCallbacks } from './helpers';
+import { Component } from 'react';
 
 modulejs.define(
   'HBWEntityTask',
   ['React', 'ReactDOM', 'jQuery', 'HBWForm', 'HBWTestForm', 'HBWError', 'HBWPending',
     'HBWFormDefinition'],
   (React, ReactDOM, jQuery, Form, TestForm, Error, Pending, FormDefinition) => {
-    const EntityTask = React.createClass({
-      displayName: 'HBWEntityTask',
-
-      getInitialState () {
-        return {
-          error:     null,
-          loading:   true,
-          pending:   null,
-          form:      null,
-          collapsed: this.props.collapsed || false
-        };
-      },
+    class EntityTask extends Component {
```

Comment:
This component had name `HBWEntityTask` but it is `EntityTask` now because React builds component name from name of the class.

Reply:
Than add everywhere `HBW` prefix?

Reply:
Let's just go back to what we had :)